dbt_semantic_interfaces/validations/unique_valid_name.py

```diff
@@ -69,9 +73,9 @@ def check_valid_name(name: str, context: Optional[ValidationContext] = None) ->
             issues.append(
                 ValidationError(
                     context=context,
-                    message=f"Invalid name `{name}` - names should only consist of lower case letters, numbers, "
-                    f"and underscores. In addition, names should start with a lower case letter, and should not end "
-                    f"with an underscore, and they must be at least 2 characters long.",
+                    message=f"Invalid name `{name}` - names may only contain lower case letters, numbers, "
+                    f"and underscores. Additionally, names must start with a lower case letter, cannot end "
+                    f"with an underscore, cannot contain dunders (__), and must be at least 2 characters long.",
```

Comment:
nit: DUNDER and __ might not render in an obvious way.
```suggestion
                    f"with an underscore, cannot contain dunders (double underscores, or __), and must be at least 2 characters long.",
```

Reply:
Added the suggested change via a fixup to "Improve validation error message for invalid names" commit, and force pushed.